### PR TITLE
Document the Media Viewer app

### PR DESCRIPTION
**Pull request by settermjd**
This change documents the core aspects of the Media Viewer app, originally begun in #1515. This is in follow-up to last Friday's documentation discussion meeting.

### CODE REVIEW

File: modules/user_manual/pages/files/media_viewer_app.adoc [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+IMPORTANT: Please remove the gallery and files_videoplayer apps before installing it.
+
+It supports the following functionality:
+
+* *Image*: _preview_, _zoom_, _rotation_, and _download_
+* *Video*: _playback_ and _download_
+
+NOTE: The app will support paginating through all media files in the current directory, even if only one media file was chosen to be previewed.
+
+== Features
```
Comment (pmaier1) [inline modules/user_manual/pages/files/media_viewer_app.adoc]: The part below seems to be copied from the gallery info.xml description. Some parts have changed. Please use the description from https://marketplace.owncloud.com/apps/files_mediaviewer here.

File: modules/user_manual/pages/files/media_viewer_app.adoc [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+* Per album design, description and copyright statement.
+* Sort images by name or date added.
+* Support for a large selection of media types (dependent on the server setup).
+* Switch to Gallery from any folder in files and vice-versa.
+* Upload and organise images and albums straight from the app.
+
+== Supported File Formats
+
+* Video: {mp4-url}[MP4], {ogg-url}[Ogg], {webm-url}[WebM]
+* Image: All image formats {browser-image-support-url}[supported by your browser].
```
Comment (pmaier1) [inline modules/user_manual/pages/files/media_viewer_app.adoc]: - The supported **video** formats depend on the **user's browser**
- The supported **image** formats depend on the **server capabilities**

**Comment by settermjd** [comment]
@pmaier1, I've made some additional changes, specifically linking to the app in the Marketplace, and covering the additional file formats that can be enabled.

**Comment by settermjd** [comment]
@pmaier1 / @micbar, time-permitting, please let me know if any further changes or additions are required.

File: modules/user_manual/pages/files/media_viewer_app.adoc [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+* Mobile support
+
+== Supported File Formats
+
+* The supported video formats depend on the user's browser
+* The supported image formats depend on the server capabilities
+
+== Configuring the Media Viewer App
+
+By default, the Media Viewer app supports previewing eight file formats; these are:
```
Comment (pmaier1) [inline modules/user_manual/pages/files/media_viewer_app.adoc]: Where does this information come from? The media viewer does not play music (mp3) or display text (txt/markdown).

File: modules/user_manual/pages/files/media_viewer_app.adoc [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+In addition to these, support for a number of additional file formats can be enabled; these are:
+
+* Font files
+* Adobe Illustrator, Photoshop, PDF, and Postscript files
+* SVG and TIFF files
+* Microsoft Office 2003 & 2007
+* OpenDocument and StarOffice files
+
+To enable these, an ownCloud or systems administrator needs to add the relevant configuration to the `enabledPreviewProviders` key in `config/config.php`.
```
Comment (pmaier1) [inline modules/user_manual/pages/files/media_viewer_app.adoc]: Everything below is actually admin doc, but this file is part of the user documentation

File: modules/user_manual/pages/files/media_viewer_app.adoc [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+* Image and video download straight from the slideshow
+* Native SVG support
+* Mobile support
+
+== Supported File Formats
+
+* The supported video formats depend on the user's browser
+* The supported image formats depend on the server capabilities
+
+== Configuring the Media Viewer App
```
Comment (pmaier1) [inline modules/user_manual/pages/files/media_viewer_app.adoc]: Everything below is actually admin doc, but this file is part of the user documentation

File: modules/user_manual/pages/files/media_viewer_app.adoc [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+:media-viewer-app-url: https://marketplace.owncloud.com/apps/files_mediaviewer
+
+image:media-viewer-app/app-overview.gif[]
+
+== Introduction
+
+The {media-viewer-app-url}[Media Viewer app] is a lightweight viewer for pictures and videos which integrates with the files app, and is released under the GPLv2. 
+It replaces the gallery and files_videoplayer apps, which have now been deprecated, and supports the same basic feature set as the deprecated apps.
+
+IMPORTANT: Please remove the gallery and files_videoplayer apps before installing it.
```
Comment (pmaier1) [inline modules/user_manual/pages/files/media_viewer_app.adoc]: This information should be part of the admin doc.

File: modules/user_manual/pages/files/media_viewer_app.adoc (hunk 3 above)
Comment (settermjd) [reply modules/user_manual/pages/files/media_viewer_app.adoc]: That information's drawn directly from `OC\PreviewManager::getEnabledDefaultProvider()`. I'll remove references to the providers that aren't applicable to the Media Viewer.

Comment (pmaier1) [reply modules/user_manual/pages/files/media_viewer_app.adoc]: Thanks! Afaik these are just the previews that `core` can create. The media viewer "only" displays video and image formats.

Comment (settermjd) [reply modules/user_manual/pages/files/media_viewer_app.adoc]: okay.

**Comment by settermjd** [comment]
@pmaier1, the relevant changes have been made.

**Pull request merged**